aspnetcore/fundamentals/servers/kestrel/options.md

```diff
@@ -120,7 +119,7 @@ Gets or sets the [keep-alive timeout](https://tools.ietf.org/html/rfc7230#sectio
 
 ### Maximum client connections
 
-<xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerLimits.MaxConcurrentConnections>
+<xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerLimits.MaxConcurrentConnections>  
```

Comment:
Is there any more robust syntax available? These spaces are likely to be trimmed by editors or well meaning contributors.